src/Nethermind/Nethermind.Evm/TransactionProcessing/TransactionProcessor.cs

```diff
@@ -346,7 +346,7 @@ senderBalance < (UInt256)transaction.GasLimit * transaction.MaxFeePerGas + value
                     : new CodeInfo(machineCode);
 
                 ExecutionType executionType =
-                    transaction.IsContractCreation ? ExecutionType.Create : ExecutionType.Call;
+                    transaction.IsContractCreation ? ExecutionType.Create : ExecutionType.Transaction;
```

Comment:
Looks like a potentially big change?

Reply:
there are only two use case of ExecutionType executionType.
- tracer.ReportAction - there use of  ExecutionType.Call and ExecutionType.Transaction map to the same values in traces, the only difference would be in IncludeInTrace 
- executionType.IsCreate() - this also behaves same for ExecutionType.Call and ExecutionType.Transaction